internal/drivers/amazon/driver.go

```diff
@@ -384,6 +384,21 @@ func (p *config) Logs(ctx context.Context, instanceID string) (string, error) {
 	return string(decoded), nil
 }
 
+func (p *config) SetTags(ctx context.Context, instanceID string,
+	tags map[string]string) error {
+	in := &ec2.CreateTagsInput{
```

Comment:
Likewise, if tags are empty we should return immediately

Reply:
Added the check in manager.go